### ISSUE TITLE
Trouble with 3.10.105-0-pine64-longsleep

### ISSUE THREAD

**Issue by texadactyl** [longsleep/linux-pine64#59]
Started with: https://www.stdin.xyz/downloads/people/longsleep/tmp/pine64-images/ubuntu/xenial-pine64-bspkernel-20161218-1.img.xz
Using a 64GB MicroSD with a Pine64+ 2GB RAM and an HDMI display 1080p
Never had any trouble before in the last 12 months

====================================

On staging system
[kernel_bug.tar.gz](https://github.com/longsleep/linux-pine64/files/1201363/kernel_bug.tar.gz)
[kernel_bug.tar.gz](https://github.com/longsleep/linux-pine64/files/1201366/kernel_bug.tar.gz)

,
     Make sure that my target is /dev/sdc
     Use fdisk to clear out any existing partitions on /dev/sdc
     sudo dd if=/home/elkins/Downloads/xenial-pine64-bspkernel-20161218-1.img of=/dev/sdc bs=4M oflag=sync
     sync; sync; sync
     sudo eject /dev/sdc

====================================

Inserted MicroSD and powered up fine.

Image.version: 3.10.104-2-pine64-longsleep

installed ssh & permited root login

resize_rootfs.sh
reboot
ok

pine64_update_uboot.sh
reboot
ok

pine64_update_kernel.sh
Checking for update ...
Downloading Linux Kernel ...
######################################################################## 100.0%
Downloading signature ...
######################################################################## 100.0%
Downloading public key ...
######################################################################## 100.0%
Verifying signature ...
[GNUPG:] SIG_ID uobXqzSdyGQtD48Cgolff09/5Lg 2017-03-11 1489251421
[GNUPG:] GOODSIG 7C03F26B090EF0DB Simon Eisenmann <simon@struktur.de>
[GNUPG:] VALIDSIG 6FC96963C3D26AA8800974A17C03F26B090EF0DB 2017-03-11 1489251421 0 4 0 1 2 00 9DDB6CAD455D63112CAD26D8CA499C410B3C3354
Extracting ...
Fixing up ...
Running postinst for 3.10.105-0-pine64-longsleep ...
Done - you should reboot now.

THEN, rebooted
Cannot get to a tty.
Monitor status: disconnected (red).
ssh client can no longer connect.

Uncable power, re-cable power.
Still the same. Won't boot.

Go back to staging system

Image.version on MicroSD: 3.10.105-0-pine64-longsleep

Attached should be a tar.gz of final /var/log directory tree.

**Comment by texadactyl** [comment]
Fell back to original kernel 3.10.104-2-pine64-longsleep by downloading  https://www.stdin.xyz/downloads/people/longsleep/tmp/pine64-images/ubuntu/xenial-pine64-bspkernel-20161218-1.img.xz again and reinstalling.  Post-install, updated all of the apt repo packages to current level, added the Mate desktop, and setiathome running on 2 cores.  Cool temp (~65C) and stable.

uname -a >> 3.10.104-2-pine64-longsleep Thu Dec 15 21:46:07 CET.

I can stay at this kernel level without issues.

**Comment by longsleep** [comment]
It means that your power supply is insufficient. Later kernels enable more stuff an thus can result in crash on boot due to fatal voltage drop.

**Comment by texadactyl** [comment]
My power supply (5V, 2.5A) is fine; same as for my Raspberry Pi.  However, I noticed that when I moved the power adapter A/C connection to a different wall socket connection, 105 started to boot okay.  

Now, why would 104 boot ok but 105 wouldn't? They are both in the 3.10 family, very little difference.  

Anyway, time to get out my multimeter.   I have a flakey wall socket.

Sorry to have been a bother.
